src/validation-errors/__fixtures__/additionalProperties/data.json

```diff
@@ -0,0 +1,5 @@
+{
+  "foo": "bar",
+  "bar": 2,
+  "baz": []
+}
```

Comment:
New line plz 😎. And, use prettier 😛 I will update the hook soon.

Reply:
fixed c3e9a19